lib/fp.js

```diff
@@ -2,32 +2,29 @@
 
 const { safeCallback } = require('./callbacks');
 
-const partial = (
-  // Partially apply arguments to function
-  fn, // function
-  ...args // array, argumants
-  // Returns: function
-) => (...rest) => fn(...args.concat(rest));
-
-const omap = (
-  // Map object fields with provided function
-  mapFn, // funtion, to appy to every field value
-  obj // object, which fields used for mapping
-  // Returns: object, with same reference but with transformed fields
-) => {
+// Partially apply arguments to function
+//   fn - function
+//   args - array, argumants
+// Returns: function
```

Comment:
This should probably be `function(...args), args - arguments`  (Also applies in other places)